src/website/views/custom-elements/spinner.pug

```diff
@@ -0,0 +1,82 @@
+---
+title: Spinner
+scripts:
+  - js/ce/ux-chi-ce
+---
+
+p.-text Spinners are animated circles that let users know when information is saving or loading.
+
+h3 Examples
+p.-text To render a spinner, use <code>chi-spinner</code> tag.
+
+h3 Backdrop
+p.-text Wrap spinners in <code>a-backdrop</code> to overlay content similar to a modal.
+h4 Base
+.example.-mb--3
+  .a-backdrop.-p--6(style="position:relative;height:25rem;")
+    chi-spinner(color='inverse', size='lg')
+  :code(lang="html")
+    <div class="a-backdrop">
+      <chi-spinner color="inverse"></chi-spinner>
+    </div>
+
+h4 Inverse
+p.-text Apply the class <code>-inverse</code> to render spinners on light backgrounds.
+.example.-mb--3
+  .a-backdrop.-inverse.-p--6(style="position:relative;height:25rem;")
+    chi-spinner(size='lg', color='primary')
+  :code(lang="html")
+    <div class="a-backdrop -inverse">
+      <chi-spinner></chi-spinner>
+    </div>
+
+h3 Contextual
+
+h3 Spinner Sizes
+p.-text
+  | By default, spinners render at 16x16px (<code>sm</code>).
+p.-text
+  | Apply size classes to render spinners larger or smaller: <code>xs</code>,
+  | <code>sm</code>, <code>sm--2</code>, <code>sm--3</code>, <code>md</code>, <code>lg</code>, <code>xl</code>, <code>xxl</code>.
+.example.-mb--3
+  .-p--3
+    .a-grid.-align-items--end
+      each size in ['xs', 'sm', 'sm--2', 'sm--3', 'md', 'lg', 'xl', 'xxl']
+        div.-p--2
+          chi-spinner(size=`${size}`)
+          p.-text.-text--center= size
+  :code(lang="html")
+    <!-- xs : 12x12px -->
+    <chi-spinner size="xs"></chi-spinner>
+    <!-- sm : 16x16px -->
+    <chi-spinner size="sm"></chi-spinner>
+    <!-- sm--2 : 20x20px -->
+    <chi-spinner size="sm--2"></chi-spinner>
+    <!-- sm--3 : 24x24px -->
+    <chi-spinner size="sm--3"></chi-spinner>
+    <!-- md : 32x32px -->
+    <chi-spinner size="md"></chi-spinner>
+    <!-- lg : 64x64px -->
+    <chi-spinner size="lg"></chi-spinner>
+    <!-- xl : 96x96px -->
+    <chi-spinner size="xl"></chi-spinner>
+    <!-- xxl : 128x128px -->
+    <chi-spinner size="xxl"></chi-spinner>
+
+h3 Spinner Colors
+p.-text
+  | By default, spinner color is (<code>primary</code>).
+.example.-mb--3
+  .-m--3
+    .a-grid.-align-items--center
+      each type in ['primary', 'success', 'warning', 'danger', 'muted']
+        div.-p--2
+          chi-spinner(color=`${type}`, size='md')
+            p.-text(class=`-text--${type}`)
```

Comment:
This p element should be outside chi-spinner and also should be below rather than above the element.